_data/sidebars/lb4_sidebar.yml

```diff
@@ -103,13 +103,18 @@ children:
   output: 'web, pdf'
   children:
 
-  - title: 'Defining and validating the API'
-    url: Defining-and-validating-the-API.html
+  - title: 'Defining the API using code-first approach'
```

Comment:
`Defining API's using code-first approach` might be better. `Defining the API ...` seems odd.

Reply:
+1. As a helpful comment though, never use apostrophe to make something plural :p